### PR TITLE
Coverage: Use find and file module instead of shell command

### PR DESCRIPTION
**Pull request by akash2237778**
Update tests/get-coverage.yml
Coverage: Use find and file module instead of shell command #177

### CODE REVIEW

**Comment by akash2237778** [comment]
@tyll  Sir, Review this once then I will close previous PR #177

**Comment by akash2237778** [comment]
@tyll 
Sir, this time updated my branch without creating a new PR. All thanks to docs send by you.
Kindly run the checks and merge PR.

File: tests/get-coverage.yml [hunk 1]
```diff
@@ -37,8 +37,18 @@
         state: absent
         path: "{{ coverage_file }}"
 
+    - name: find coverage files to delete
+      find:
+        path: "{{ playbook_dir }}"
```
Comment (tyll) [inline tests/get-coverage.yml:42]: This path seems to be wrong. It needs to be the remote user's home directory.

Comment (akash2237778) [reply tests/get-coverage.yml]: @tyll 
Sir, I have done the required changes pls have a look. And suggest if need any more changes.

**Comment by akash2237778** [comment]
Sir, pls have a look I've done the required changes

**Comment by tyll** [comment]
Thank you. Please also update the author information in the git commit itself as described in https://stackoverflow.com/questions/3042437/how-to-change-the-commit-author-for-one-specific-commit

**Comment by akash2237778** [comment]
> Thank you. Please also update the author information in the git commit itself as described in https://stackoverflow.com/questions/3042437/how-to-change-the-commit-author-for-one-specific-commit

Done Sir
Thanks for the opportunity.

**Comment by akash2237778** [comment]
Done Sir

**Comment by tyll** [comment]
[citest commit:43014f318ea8ce3b8fde4c3a3848d450db2eafee]

**Pull request merged**